### PR TITLE
Phani/em 1703 text highlight rotate

### PR DESCRIPTION
**Pull request by phaniperla**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: e2e/src/pages/app.po.ts [hunk 1]
```diff
@@ -41,7 +38,7 @@ export class AppPage {
         this.clickElement(by.css('label[for="download-btn-toggle"]')),
         this.clickElement(by.css('label[for="navigate-btn-toggle"]')),
         this.clickElement(by.css('label[for="print-btn-toggle"]')),
-        this.clickElement(by.css('label[for="rotate-btn-toggle"]')),
+        // this.clickElement(by.css('label[for="rotate-btn-toggle"]')),
```
Comment (velimd) [inline e2e/src/pages/app.po.ts]: why not remove these comments completely?

File: e2e/src/pages/app.po.ts [hunk 2]
```diff
@@ -69,6 +66,10 @@ export class AppPage {
     await browser.wait(until.presenceOf(element(by.css('div[class="page"'))), 15000, 'PDF viewer taking too long to load');
   }
 
+  async waitForImageToLoad() {
```
Comment (jai-santhosh) [inline e2e/src/pages/app.po.ts]: Do we need this duplicate code ? waitforImageToLoad I assume you are waiting in the image viewer and that has the same class as pdf ? ( .page) ?

File: e2e/src/pages/app.po.ts (hunk 1 above)
Comment (phaniperla) [reply e2e/src/pages/app.po.ts]: Will take care that in code optimisation

File: e2e/src/pages/app.po.ts (hunk 2 above)
Comment (phaniperla) [reply e2e/src/pages/app.po.ts]: Am sorry I forgot to remove that duplicate code

Comment (phaniperla) [reply e2e/src/pages/app.po.ts]: deleted unused code

Comment (jai-santhosh) [reply e2e/src/pages/app.po.ts]: great

**Pull request merged**